_aggregations/metric/average.md

```diff
@@ -9,9 +9,20 @@ redirect_from:
 
 # Average aggregations
 
-The `avg` metric is a single-value metric aggregations that returns the average value of a field.
+The `avg` metric is a single-value metric that returns the average value of a field.
 
-The following example calculates the average of the `taxful_total_price` field:
+## Parameters
+
+The `avg` aggregation takes the following parameters: 
+
+| Parameter | Required/Optional | Data type       | Description |
+| :--       | :--               | :--            | :--         |
+| `field`   | Required          | Numeric         | The field for which the average is computed    |
+| `missing` | Optional          | Numeric         | Value to assign missing instances of the field |
+
+## Example
+
+This example calculates the average of the `taxful_total_price` field in the commerce sample data:
 
 ```json
 GET opensearch_dashboards_sample_data_ecommerce/_search
```

Comment:
The convention seems to be to label the code block as JSON even though the GET statement isn't. Is that correct?

Reply:
@dwelsch-esi Yes. We have a custom lexer that highlights methods.